android/app/src/main/java/app/organicmaps/widget/placepage/EditBookmarkFragment.java

```diff
@@ -103,6 +105,20 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState)
     initToolbar(view);
   }
 
+  @Override
+  public void onStart()
+  {
+    super.onStart();
+
+    // focus name and show keyboard for "Unknown Place" bookmark
+    if (mBookmark != null && mBookmark.getName().equals(getString(R.string.core_placepage_unknown_place)))
+    {
+      mEtName.requestFocus();
+      mEtName.selectAll();
+      WindowCompat.getInsetsController(requireActivity().getWindow(), mEtName).show(WindowInsetsCompat.Type.ime());
```

Comment:
Thanks! Why is this line needed? What does it do?

Reply:
Because the activity is just starting, showing the keyboard with `app.organicmaps.util.InputUtils.showKeyboard()` does not work. This is the [recommended way](https://developer.android.com/develop/ui/views/touch-and-input/keyboard-input/visibility#ShowReliably) of dealing with that.

Reply:
its good to add a comment about this into the code

Reply:
Okay, I added a comment!